core/src/main/resources/hudson/model/View/sidepanel_pl.properties

```diff
@@ -22,7 +22,7 @@
 
 Build\ History=Historia zada\u0144
 Check\ File\ Fingerprint=Sprawd\u017A Odcisk Palca Pliku
-Delete\ View=Usu\u0144 Widok
+Delete\ View=Usu\u0144 widok
```

Comment:
In Polish only first word is capitalized except names of the companies organisations products or roles. Hope its clear now.